moped-editor/src/views/projects/projectView/ProjectPhaseForm.js

```diff
@@ -118,31 +148,45 @@ const ProjectPhaseForm = ({
     }
   }, [phase_end, defaultValues, setValue]);
 
-  if (mutationState.error) {
-    console.error(mutationState.error);
+  /**
+   * Defaults phase_start to today if current phase is true and there is no phase_start
+   */
+  useEffect(() => {
+    if (is_current_phase === true && phase_start === null) {
+      const today = new Date();
+      today.setHours(0, 0, 0, 0);
+
+      setValue("phase_start", today);
+    }
+  }, [is_current_phase, phase_start, setValue]);
+
+  if (mutationState.error || addStatusUpdateState.error) {
+    console.error("Phase mutation error", mutationState.error);
+    console.error("Status update mutation error", addStatusUpdateState.error);
     return (
       <Grid container spacing={2}>
-        <Grid item xs={12}>
-          <Alert severity="error">
-            Something went wrong. Refresh the page to try again.
-          </Alert>
-        </Grid>
+        {mutationState.error ? (
```

Comment:
Here is a screenshot of what this would look like if both mutations errored for some reason. I don't think we've handled errors coming from multiple submissions at once. Looking for feedback! 🙏
<img width="624" alt="Screenshot 2024-01-26 at 4 13 23 PM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/b3fefe59-9102-4135-a5dd-87bb6060655d">

Reply:
Thanks for thinking through these error cases! some of the branching logic might get ugly, but I'm wondering what you think about combining the two mutations into a single request on submit—so as to handle this as an all-or-nothing operation?

Reply:
I think that combining into one mutation is the right call. I'll make that change and get it pushed up!